helpdesk_mgmt_stage_validation/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+from .validate_utils import validate_stage_fields
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    @api.constrains("stage_id")
+    def _validate_stage_fields(self):
+        validate_stage_fields(self)
```

Comment:
Here is the only location where you use the function `validate_stage_fields`.
For me it's better to have directly here the function: it's more easy to read/maintain, and extract the function have no interest.